liblouis/liblouis.h.in

```diff
@@ -319,6 +319,35 @@ LIBLOUIS_API
 void EXPORT_CALL
 lou_setLogLevel(logLevels level);
 
+/* ========================= Sort-of private API ========================= */
+
+/**
+ * Definitions of braille dots
+ */
+typedef enum BrailleDots {
+	LOU_DOT_1 = 0X01,    /** dot 1 */
+	LOU_DOT_2 = 0X02,    /** dot 2 */
+	LOU_DOT_3 = 0X04,    /** dot 3 */
+	LOU_DOT_4 = 0X08,    /** dot 4 */
+	LOU_DOT_5 = 0X10,    /** dot 5 */
+	LOU_DOT_6 = 0X20,    /** dot 6 */
+	LOU_DOT_7 = 0X40,    /** dot 7 */
+	LOU_DOT_8 = 0X80,    /** dot 8 */
+	LOU_DOT_9 = 0X100,   /** virtual dot 9 */
+	LOU_DOT_10 = 0X200,  /** virtual dot A */
+	LOU_DOT_11 = 0X400,  /** virtual dot B */
+	LOU_DOT_12 = 0X800,  /** virtual dot C */
+	LOU_DOT_13 = 0X1000, /** virtual dot D */
+	LOU_DOT_14 = 0X2000, /** virtual dot E */
+	LOU_DOT_15 = 0X4000, /** virtual dot F */
+	LOU_DOT_16 = 0X8000  /** virtual dot 0 */
```

Comment:
Would `LOU_DOT_NONE` work?

Reply:
Maybe `LOU_DOTS`? Because the bit will be set for all dot patterns.

Reply:
I renamed it to `LOU_DOT_NONE`. Hope that works for you.

Reply:
I don't like it. But I don't wanna be a kill-joy. If you don't like `LOU_DOTS`, it's one against one.

Reply:
I thought `LOU_DOT_NONE` fit the meaning of the value better, maybe I misunderstand what the value really means. I don't really care that much

Reply:
OK I think it's pretty important to understand what it means if you want to name it. What does it mean according to you? :-)

Reply:
`LOU_DOTS` seems so generic, so /nichtssagend/ (empty, meaningless)

Reply:
I thought it was some kind of sentinel to indicate an impossible dots value

Reply:
Not sure what sentinel means. But it's a bit that is always true when a widechar represents a dot pattern. (Maybe that's what you meant.)

It's not a very useful bit I think, because in almost all cases in the program you know that it is a dot pattern, so if the bit is not set, there is a bug. The exception is lou_dotsToChar, which accepts either a dot pattern or a Unicode character in the U+28xx range. Also it doesn't make sense to use this bit to differentiate between dot patterns and characters, because this way you make it impossible to handle characters in the range U+8000 - U+EFFF.

Anyway, `LOU_DOTS` stands for: "if this bit is true, it is a dot pattern". `LOU_DOT_1` stands for: "if this bit is true, the dot pattern contains dot 1". `LOU_DOT_NONE` presumably would stand for "if this bit is true AND all the other bits are false, it is a blank dot pattern", which makes less sense to me because it would be about the whole widechar value, not a single bit (like is the case for all the other constants).

Reply:
Aha, OK, then I guess the name makes sense, thanks